docs/testcases/UI/System_Management/2232-2444_NAT_Gateway/2232_9.md

```diff
@@ -22,9 +22,9 @@ Test verifies that
 | 5 | Click **Add Route** button | |
 | 6 | Input `<Server_name_3>` into the ***Server name:*** field |  |
 | 7 | Input `80` into the ***Port:*** field | |
-| 8 | Click **Add Port** button | **Add** button becomes disabled |
+| 8 | Click **Add Port** button | **Add** button becomes disabled | |
```

Comment:
Remove extra `|` symbol in the end of the row

Reply:
Done